src/containers/User/UserArea.tsx

```diff
@@ -182,17 +183,23 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({ templates, dataViews }) => {
 
 const BrandArea: React.FC = () => {
   const { strings } = useLanguageProvider()
+  const { preferences } = usePrefs()
+
+  const logoUrl = preferences?.brandLogoOnDarkFileId
+    ? `${config.serverREST}/public/file?uid=${preferences.brandLogoOnDarkFileId}`
```

Comment:
`nit:` it kind of feels like the server should be giving the full url (minus domain), or maybe a helper to get the file by id, this is constructed twice in f/e

Reply:
The reason I didn't provide the whole url in the server is that the "preferences.json" file that provides it is currently manually created. The whole prefs file gets imported/exported with snapshots, but the contents is just hand-made for now. (Until we make a UI for editing prefs). And so I figured it was better to keep the prefs content to just the essentials (i.e. fileID) rather than the whole url (which could in theory change, or more likely get mistyped -- trailing slash? No slash? -- that kind of thing).

Also, we've generally used hard-coded values for endpoints in the front-end throughout, so just kept with that pattern. Ideally we should have a global endpoint map, but that seems like something for another day as I'd want to replace all hard-coded endpoint urls.

Reply:
Cool, thanks